src/customize-pop/default-applications.md

```diff
@@ -0,0 +1,83 @@
+# Default Applications
+
+Options to set the default web browser, email client, calendar application, music player, video player, and photo viewer are located in `Settings` ➞ `Default Applications`. This menu allows setting a single application as the default handler for a larger number of similar file types.
+
+![Default Applications](/images/application-settings/default-applications.png)
+
+Designate an application as the default by selecting it from the dropdown menu. If no appropriate application is installed to handle a media type, the dropdown selection will be grayed out. <!--link to installing applications section when complete-->
+
+![Select Default](/images/application-settings/select-default.png)
+
+## Set Default Applications Using the File Manager
+
+Launch the file manager and navigate to the file you want to open. Right click the file and select `Properties`.
+
+![File Properties](/images/application-settings/file-properties.png)
+
+Click the `Open With` tab and choose the appropriate program from the list.
+
+![Open With](/images/application-settings/open-with.png)
+
+## Set Default Applications Using the Terminal
+
+### Modify the Mimeapps.list File
+
+The `mimeapps.list` file contains entries associating applications with specific links and MIME file types. Application names listed in this file will be appended with `.desktop`.
+
+1. Determine your application's .desktop name using this command to list all .desktop applications:
+
+    ```bash
+    ls /usr/share/applications
+    ```
+
+This command will provide output similar to the below image. Take note of your desired default application's .desktop name, or leave this Terminal window open and start a new Terminal session to complete the next steps.
+
+![Output]()
+
+2. Open the `mimeapps.list` file using this command:
+
+    ```bash
+    nano ~/.config/mimeapps.list
+    ```
+
+4. Locate the file MIME type entry and edit it using the .desktop application name. For example, this entry requires that all .webm files are opened with VLC media player:
+
+    ```bash
+    video/webm=vlc.desktop;
+    ```
+
+If a file's MIME type is not listed in `mimeapps.list`, you can determine a file's MIME type by navigating to the file's location and using the `mimetypo` command:
+
+    ```bash
```

Comment:
These code blocks should not be indented four spaces because they are not part of a list item. (That's causing them to be double-code-blocked, and the "```bash" is showing up at the beginning.)